bedrock/firefox/urls.py

```diff
@@ -118,4 +118,6 @@
 
     url('^firefox/stub_attribution_code/$', views.stub_attribution_code,
         name='firefox.stub_attribution_code'),
+
+    page('firefox/foo', 'firefox/base-pebbles.html'),
```

Comment:
You can view the base template using this temporary URL (leaving this here so I don't forget to remove it again before merging).